### PR TITLE
Clean up / Fix JavaDoc errors

### PR DESCRIPTION
**Pull request by sigee**

### CODE REVIEW

File: src/org/zaproxy/zap/extension/spider/SpiderPanel.java [hunk 1]
```diff
@@ -142,7 +142,7 @@
 	 * <p>
 	 * Lazily initialised.
 	 * 
-	 * @see #getAddedNodesTableScrollPane()
+	 * @see #getAddedUrlsTableScrollPane()
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/spider/SpiderPanel.java]: Better to rename the method.

File: src/org/zaproxy/zap/spider/filters/ParseFilter.java [hunk 2]
```diff
@@ -62,7 +62,7 @@ public FilterResult filtered(HttpMessage responseMessage) {
 	 *
 	 * @since 2.7.0
 	 * @see #NOT_FILTERED
-	 * @see #ParseFilter(String)
+	 * @see #filtered(HttpMessage)
```
Comment (thc202) [inline src/org/zaproxy/zap/spider/filters/ParseFilter.java]: It should be `#FilterResult(String)` instead (though Java 8 does not process it correctly).

**Comment by thc202** [comment]
Guess the ZAP comments can be skipped, @kingthorin, @psiinon wdyt?

**Comment by kingthorin** [comment]
I think so, it's a non-functional change.

**Comment by kingthorin** [comment]
Please do an interactive rebase and fix-up or squash into a single "atomic" commit: https://github.com/zaproxy/zaproxy/blob/develop/CONTRIBUTING.md#what-we-zap-team-expect-from-you .

- https://www.instructables.com/id/How-to-Rebase-Interactive-With-Eclipse-eGit/
- https://www.kernel.org/pub/software/scm/git/docs/git-rebase.html#_interactive_mode
- https://wiki.eclipse.org/EGit/User_Guide#Interactive_Rebase

**Comment by thc202** [comment]
Thanks!

**Comment by thc202** [comment]
@sigee how would you like to be credited?
https://github.com/zaproxy/zap-core-help/wiki/HelpCredits

**Pull request merged**